### PR TITLE
Martial Art error fix

### PR DESCRIPTION
**Pull request by ISuckM8**
#### Summary

Bugfixes "Fixes the martial art loading error"

#### Purpose of change

#51662 screwed up the martial art JSON, now you can't load the game.

#### Describe the solution

"int skill_level = skillArray.get_int( 1 );" instead of "int skill_level = 0;"

#### Describe alternatives you've considered

Return "autolearn": [ [ "melee", "1" ] ]

#### Testing

Works

### CODE REVIEW

**Comment by ISuckM8** [comment]
> Loading code should be fixed, not the json.

Both should fix the game but I don't know how to fix the loading code so...

**Comment by ZhilkinSerg** [comment]
image

**Comment by ISuckM8** [comment]
Thanks, changed the PR.

**Comment by ISuckM8** [comment]
There we go.

File: src/martialarts.cpp [hunk 1]
```diff
@@ -261,7 +261,7 @@ void martialart::load( const JsonObject &jo, const std::string & )
     mandatory( jo, was_loaded, "initiate", initiate );
     for( JsonArray skillArray : jo.get_array( "autolearn" ) ) {
         std::string skill_name = skillArray.get_string( 0 );
-        int skill_level = 0;
+        int skill_level = skillArray.get_int( 1 );
         std::string skill_level_string = skillArray.get_string( 1 );
         skill_level = stoi( skill_level_string );
```
[the remark this answers is not in the document]
Comment (ISuckM8) [reply src/martialarts.cpp]: Thanks.

**Pull request merged**